README.md

```diff
@@ -180,6 +181,19 @@ densityplot!(plt, randn(1000) .+ 2, randn(1000) .+ 2)
 ```
 ![Density Screenshot](https://github.com/JuliaPlots/UnicodePlots.jl/raw/unicodeplots-docs/doc/imgs/2.x/densityplot1.png)
 
+#### Contour Plot
+```julia
+x = -3:.1:3
+y = -7:.1:3
+X = repeat(x', length(y), 1)
+Y = repeat(y, 1, length(x))
+g(x, y, x₀=0, y₀=-2, σx=1, σy=2) = exp(-((x - x₀) / 2σx)^2 - ((y - y₀) / 2σy)^2)
+contourplot(x, y, map(g, X, Y), levels = 3)
```

Comment:
Does it make sense to you to also support `countourplot(x, y, g)` here? I know there's usage like `surface(x, y, g)` in Plots but I don't know how widely it's used in practice.

Reply:
Good point, since we already support this for e.g. [lineplot](https://github.com/JuliaPlots/UnicodePlots.jl/blob/980d17cdc372599c87f88e502775d408f30fd38b/src/interface/lineplot.jl#L192). I will add this.